docker-java/src/test/java/com/github/dockerjava/cmd/EventsCmdIT.java

```diff
@@ -113,6 +114,31 @@ public void testEventStreamingWithFilter() throws Exception {
         }
     }
 
+    @Test
+    public void testEventStreamingWithEventTypeFilter() throws Exception {
+        assumeNotSwarm("", dockerRule);
+
+        String startTime = getEpochTime();
+        int expectedEvents = 1;
+
+        EventsTestCallback eventCallback = new EventsTestCallback(expectedEvents);
+
+        dockerRule.getClient().eventsCmd()
+            .withSince(startTime)
+            .withEventTypeFilter(EventType.CONTAINER)
+            .withEventFilter("start")
```

Comment:
since we're testing `eventType` filter, I don't think we should filter out specific events here